practice/lib/practice/digital_root.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Root do
 
   def digital_root(value) when value <= 9, do: value
+
   def digital_root(n) do
```

Comment:
Not too sure what this is trying to do, but that might not be relevant.

Reply:
I can send you the kata @apreifsteck. 
If you are asking about the reasoning for the digital_root(num), I wanted to write a recursive function that jumps out of the loop if the value on num is equal a single digit integer.